Makefile

```diff
@@ -21,6 +21,7 @@ BOOTSTRAP_CACHE="c7afb99ad"
 DISTDIR=./dist
 SNAPSHOTDIR=./snapshot
 GITTREESTATE=$(if $(shell git status --porcelain),dirty,clean)
+SYFTVERSION=$(shell go list -m all | grep github.com/anchore/syft | awk '{print $$2}')
```

Comment:
I don't know if this is any safer than what you've already done, but there's a "machine readable" flow using JSON, too, like this:

```
go list -m -json all | jq --raw-output 'select(.Path == "github.com/anchore/syft").Version'
```

Reply:
Sorry I just saw this comment. Happy to make the change if the taskrunners have jq installed. I used awk because I was more certain that would be available in the workflow by default than something like jq.